File: data/java/COUNT_NUMBER_BINARY_STRINGS_WITHOUT_CONSECUTIVE_1S_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(86);
    param0.add(75);
    param0.add(14);
    param0.add(5);
    param0.add(41);
    param0.add(35);
    param0.add(30);
    param0.add(89);
    param0.add(84);
    param0.add(53);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("COUNT_NUMBER_BINARY_STRINGS_WITHOUT_CONSECUTIVE_1S," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("COUNT_NUMBER_BINARY_STRINGS_WITHOUT_CONSECUTIVE_1S," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}

File: data/python/COUNT_NUMBER_BINARY_STRINGS_WITHOUT_CONSECUTIVE_1S_MAIN.py
if __name__ == '__main__':
    param = [
    (86,),
    (75,),
    (14,),
    (5,),
    (41,),
    (35,),
    (30,),
    (89,),
    (84,),
    (53,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("COUNT_NUMBER_BINARY_STRINGS_WITHOUT_CONSECUTIVE_1S," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("COUNT_NUMBER_BINARY_STRINGS_WITHOUT_CONSECUTIVE_1S," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
